src/libostree/ostree-repo.c

```diff
@@ -4611,11 +4618,21 @@ ostree_repo_regenerate_summary (OstreeRepo     *self,
       if (!ostree_repo_load_variant (self, OSTREE_OBJECT_TYPE_COMMIT, commit, &commit_obj, error))
         goto out;
 
-      g_variant_builder_add_value (refs_builder, 
+      g_variant_dict_init (&commit_metadata_builder, NULL);
+
+      /* Forward the commit’s timestamp if it’s valid. */
+      g_variant_get_child (commit_obj, 5, "t", &commit_timestamp);
```

Comment:
You want `ostree_commit_get_timestamp()` - otherwise we won't do the big-endian swapping.  (Keep this in mind for all ostree gvariant usage)